Extract content out of signed message when does the signature verific…

hnategh opened the pull request:
…ation

As the php openssl extension supports content extraction from signed message, I thought it would be very handy to fetch content after verification and add it to the result object. I know there's a extractSignedContents within this class which does quite the same thing, but since that function requires binary openssl, it would not be an option for those who want to use only openssl extension.

yunosh commented:
I understand the reasoning, but it still doesn't make sense to extract the data in the verification method, especially if there is an extract method. Instead, you should rewrite the extract method to use the openssl extensions instead of the binary.

yunosh commented:
Do you plan to update your pull request? Do you need any help with this?

hnategh commented:
Hi Jan,
Thank you for your reply!
Yes, you are absolutely right and it make sense to rewrite the extractSignedContents. I reverted the previous changes and rewrote the method with openssl extensions, but since there was some indentation issue the commit I would like to close this request and open new pull request with changes.

Please have a look at #223

Thank You!
